### PR TITLE
DUMPER: Add feature to extract data from ISO9660 and hybrid disk images

### PR DESCRIPTION
**Pull request by Darkhood148**

### CODE REVIEW

File: devtools/dumper-companion.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
 from typing import Any
 
 import machfs
+import pycdlib
```
Comment (rvanlaar) [inline devtools/dumper-companion.py:34]: Update the documentation at the top of the file. 
Specifically add `pycdlib` to the prerequisites line.

**Review by rvanlaar — CHANGES_REQUESTED** [review]
Code itself looks alright.

Improve your commit messages by reviewing our commit guidelines: https://wiki.scummvm.org/index.php/Commit_Guidelines 
Also read the commit log for the dumpercompanion for a general feeling what's expected.

What's the reason behind a parser_hfs, parser_iso9660 and parser_hybrid?
The answer to this question should have been clear from the commit messages.

**Comment by Darkhood148** [comment]
@rvanlaar I have changed the commit messages and added pycdlib to the prerequisites line. Can you review it please?

File: devtools/dumper-companion.py [hunk 2]
```diff
@@ -392,7 +596,7 @@ def extract_partition(args: argparse.Namespace, vol) -> int:
             folders.append((upath, obj.mddate - 2082844800))
             continue
 
-        print(upath)
+        # print(upath)
```
Comment (einstein95) [inline devtools/dumper-companion.py:599]: Could this be changed to an "if not silent: print(upath)"? Otherwise nothing is printed if not in silent mode

**Comment by rvanlaar** [comment]
For the latest commit, the version check. I'm missing important information. Something along the lines of:

- library doesn't implement encoding parameter.
- link to PR
- code can be removed when PR is merged

**Comment by Darkhood148** [comment]
@rvanlaar I have made the changes.

**Comment by rvanlaar** [comment]
The changes look good. Will need to test it though with a mac and regular iso.

For the future: think about how python types could have helped:
`ExtensionTypes = Literal["udf", "joliet", "udf"]

**Comment by sev-** [comment]
Thank you, merging

**Pull request merged**